### DISCUSSION TITLE
Receive share from another app

### DISCUSSION THREAD

**Discussion by fessor10** [expo/expo#23548] [Archived: Questions] [answered]
What package or implementation can one use to receive share from another app? Like how I can share a tweet from Twitter to a WhatsApp status.

Please Kindly help me with the answers if you know. 

@amandeepmittal @raajnadar

**Comment by raajnadar** [comment]
This is not expo specific thing you can try this https://github.com/meedan/react-native-share-menu

**Reply by fessor10** [reply]
Thank you @raajnadar 

Doesn't seem like the package is still actively maintained and not sure too if this will work with expo.

**Comment by amandeepmittal** [comment]
Hey @fessor10, as @raajnadar, you can use a library like [meedan/react-native-share-menu](https://github.com/meedan/react-native-share-menu) or can search for more similar libraries on reactnative.directory: https://reactnative.directory/?search=share

**Reply by fessor10** [reply]
Hi @amandeepmittal the library suggested doesn't seem like what will work with expo. I'm guessing expo doesn't support this at the moment. 

Thank you for helping.

**Reply by raajnadar** [reply, marked as answer]
Every package cannot be added to the expo core to work with Expo Go, you need a config plugin from library or use expo dev client and do the native changes manually
